src/ragger/backend/interface.py

```diff
@@ -50,6 +54,11 @@ def __init__(self, firmware: Firmware, log_apdu_file: Optional[Path] = None):
         self.logger = get_default_logger()
         self.apdu_logger = get_apdu_logger()
 
+        if whitelisted_status:
+            self.whitelisted_status = whitelisted_status
+        else:
+            self.whitelisted_status = []
+
```

Comment:
If previous suggestion accepted:
```suggestion
        self.whitelisted_status = whitelisted_status
 ```

Reply:
:heavy_check_mark: